packages/styletron-engine-atomic/src/client/client.js

```diff
@@ -199,8 +199,22 @@ class StyletronClient implements StandardEngine {
   }
 
   renderFontFace(fontFace: FontFaceObject): string {
-    const key = declarationsToBlock(fontFace);
-    return this.fontFaceCache.addValue(key, fontFace);
+    let result = "";
+    if (Array.isArray(fontFace)) {
```

Comment:
based on the `FontFaceObject` type, it is always an [object](https://github.com/styletron/styletron/blob/master/packages/styletron-standard/src/style-types.js#L765). would be best to properly type the expected incoming values

Reply:
I've updated the param type to allow `Array` of `FontFaceObject` and `string`.